.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+        "editor.tabSize": 8
+}
```

Comment:
Added this by accident?

You can add .vscode/ to .git/info/exclude to avoid stuff like this in the future